package.json

```diff
@@ -46,7 +46,7 @@
     "@babel/cli": "^7.8.4",
     "@babel/core": "^7.8.4",
     "@babel/plugin-transform-runtime": "^7.8.3",
-    "@babel/preset-env": "^7.8.4",
+    "@babel/preset-env": "^7.8.7",
```

Comment:
Could you split this change into a separate commit and explain the change?

Reply:
done